spec/jobs/evss/disability_compensation_form/submit_form_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EVSS::DisabilityCompensationForm::SubmitForm, type: :job do
+  let(:user) { FactoryBot.create(:user, :loa3) }
+  let(:form_json) { { data: "I'm a form" }.to_json }
+
+  describe '.start' do
+    it 'queues up the submit job and starts the workflow' do
+      expect do
+        EVSS::DisabilityCompensationForm::SubmitForm.start(user, form_json)
```

Comment:
It might be more readable to use `described_class` here — there are enough different references to `EVSS::DisabilityCompensationForm` classes in these tests that it's not immediately obvious which is which without reading closely